packages/contracts-bedrock/src/L1/OptimismPortal2.sol

```diff
@@ -516,4 +521,28 @@ contract OptimismPortal2 is Initializable, ResourceMetering, ISemver {
     function numProofSubmitters(bytes32 _withdrawalHash) external view returns (uint256) {
         return proofSubmitters[_withdrawalHash].length;
     }
+
+    /// @notice Sets static configuration options for the L2 system. Only the SystemConfig contract can call this
```

Comment:
We should revert this change out of the portal2 and generally not make diffs to it as its owned by the proofs team. We will handle reconciling the diffs later